Automattic/wp-calypso
# BBE Landing: Update EN showcase images

Related to pdh1Xd-1DL-p2#comment-2228

## Proposed Changes

* Updates the images of the site build showcase on the BBE landing page.
* Also updates the document title to `Let us build your site`.
<img width="244" alt="image" src="https://user-images.githubusercontent.com/5436027/230844230-e9681893-53ed-48c0-b331-3b5ecd28432b.png">


## Testing Instructions



* Go to `/setup/site-setup/difmStartingPoint?siteSlug=`.
* Confirm that the new images are displayed in the slide show.
* Confirm that the document title has been updated.

## Pre-merge Checklist
